examples/solvers/douglas_rachford_primal_dual_heron.py

```diff
@@ -76,9 +75,9 @@ def print_objective(x):
 
 # Solve
 x = space.zero()
-odl.solvers.douglas_rachford_pd(x, prox_f, prox_cc_g, lin_ops,
+odl.solvers.douglas_rachford_pd(x, f, g, lin_ops,
                                 tau=tau, sigma=sigma, niter=20, lam=lam,
-                                callback=print_objective, prox_cc_l=prox_cc_l)
+                                callback=print_objective, l=l)
```

Comment:
When I see this, I also don't like the `l` parameter of the solver. But it's kind of hard to come up with a better name. Something more telling perhaps, like `inf_conv` or, slightly longer, `inf_conv_funcs`?